Black-Market/Black-Market/Network/Services/UserNetworkService.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UserService.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+
+import Foundation
+import Combine
+
+class UserNetworkService: NetworkService, UserNetworkInterface {
+    func signUp(params: SignUpRequest) -> AnyPublisher<DefaultResponse, Error> {
+        getData(from: SignUpEndpoint(params: params))
+            .decode(as: DefaultResponse.self)
+            .eraseToAnyPublisher()
+    }
+    
+    func signIn(params: [String : Any]) -> AnyPublisher<UserDataResponse, Error> {
```

Comment:
we should replicate the solution in the signUp method here as well

Reply:
Done